account_invoice_inter_company/models/account_move.py

```diff
@@ -34,6 +34,11 @@ def _find_company_from_invoice_partner(self):
     def _post(self, soft=True):
         """Validated invoice generate cross invoice base on company rules"""
         res = super()._post(soft=soft)
+        if not self.env.context.get("queued"):
```

Comment:
Use a more specific context key for avoiding conflicts and do the check here:

```suggestion
        if not self.env.context.get("account_invoice_inter_company_queued"):
```